torch/_torch_docs.py

```diff
@@ -2220,10 +2218,63 @@ def merge_dicts(*dicts):
 
 Example::
 
-    >>> torch.conj(torch.tensor([-1 + 1j, -2 + 2j, 3 - 3j]))
+    >>> torch.conj_physical(torch.tensor([-1 + 1j, -2 + 2j, 3 - 3j]))
     tensor([-1 - 1j, -2 - 2j, 3 + 3j])
 """.format(**common_args))
 
+add_docstr(torch.conj,
+           r"""
+conj(input) -> Tensor
+
+Returns a view of :attr:`input` with the conjugate bit set to the negated value of :attr:`input`'s
+conjugate bit. If :attr:`input` has a non-complex dtype, this function just returns :attr:`input`.
+
+.. note::
+    :func:`torch.conj` performs a lazy conjugation, but the actual conjugated tensor can be materialized
+    using :func:`torch.resolve_conj`.
+
+.. warning:: In the future, :func:`torch.conj_physical` may return a non-writeable view for an :attr:`input` of
+             non-complex dtype. It's recommended that programs not modify the tensor returned by :func:`torch.conj_physical`
+             when :attr:`input` is of non-complex dtype to be compatible with this change.
+
+Args:
+    {input}
+
+Example::
+
+    >>> x = torch.tensor([-1 + 1j, -2 + 2j, 3 - 3j])
+    >>> x.is_conj()
+    False
+    >>> y = torch.conj(x)
+    >>> y.is_conj()
```

Comment:
Show y to make clear the operation "appears" to be performed

Reply:
hmm but the print takes into account the conjugate bit, so it could be misleading to print y because it might appear to the user that the memory is being updated

Reply:
A comment can be added to explain what's going on; printing after an operation shouldn't be misleading/confusing.